Tutorials/CMakeLists.txt

```diff
@@ -49,3 +49,4 @@ if(PLATFORM_WIN32 OR PLATFORM_LINUX OR PLATFORM_MACOS)
     add_subdirectory(Tutorial25_StatePackager)
     add_subdirectory(Tutorial26_StateCache)
 endif()
+add_subdirectory(Tutorial27_PostProcessing)
```

Comment:
Let's call it Tutorial27_SSR as this one is specifically for screen-space reflections.
We will have a separate tutorial for SSAO (this scene is not good for SSAO anyway)

Reply:
Discussed in Discord